addons/sms/models/mail_thread.py

```diff
@@ -240,7 +248,7 @@ def _notify_thread(self, message, msg_vals=False, notify_by_email=True, **kwargs
         self._notify_record_by_sms(message, recipients_data, msg_vals=msg_vals, **kwargs)
         return recipients_data
 
-    def _notify_record_by_sms(self, message, recipients_data, msg_vals=False,
+    def _notify_record_by_sms(self, message, recipients_data, msg_vals=False, sms_content=None,
```

Comment:
:socks: would move parameter definition with other sms parameters (aka the line below)